packages/react-core/src/components/NumberInput/examples/NumberInput.md

```diff
@@ -159,9 +159,21 @@ class UnitThresholdNumberInput extends React.Component {
     this.minValue = 0;
     this.maxValue = 10;
 
+    this.normalizeBetween = (value, min, max) => {
+      let result = value;
+      if (min !== undefined && max !== undefined) {
+        result = Math.max(Math.min(value, max), min);
+      } else if (max === undefined && value <= min) {
+        result = min;
+      } else if (min === undefined && value >= max) {
+        result = max;
+      }
+      return result;
+    };
```

Comment:
Nit: I think getting rid of the mutation and logically unneeded undefined checks in the second and third if clean this up a lot.

```suggestion
    this.normalizeBetween = (value, min, max) => {
      if (min !== undefined && max !== undefined) {
        return Math.max(Math.min(value, max), min);
      } else if (value <= min) { 
        return min;
      } else if (value >= max) { 
        return max;
      }
      return value;
    };
```